.vscode/launch.json

```diff
@@ -0,0 +1,42 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug CM7 - ST-Link",
+            "cwd": "${workspaceFolder}",
+            "type": "cortex-debug",
+            "executable": "${workspaceFolder}/build/heliguy_fmu-h747ai_cm7/nuttx",
+            "loadFiles": [
+                "${workspaceFolder}/build/heliguy_fmu-h747ai_cm7/nuttx",
+                "${workspaceFolder}/build/heliguy_fmu-h747ai_cm4/nuttx",
+            ],
+            "request": "launch",
+            "servertype": "stlink",
+            "gdbPath": "/usr/bin/gdb-multiarch",
+            "device": "STM32H747AI",
+            "interface": "swd",
+            "runToEntryPoint": "main",
+            "svdFile": "${workspaceFolder}/STM32H747_CM7.svd",
+            "v1": false,
+            "showDevDebugOutput": "both",
+            "serverArgs": [
+                "-l", "1",
+                "-m", "0",
+                "-k",
+                "-t","-s"
+            ],
+        },
+        {
+            "name": "Cortex Debug",
+            "cwd": "${workspaceFolder}",
+            "executable": "./bin/executable.elf",
+            "request": "launch",
+            "type": "cortex-debug",
+            "runToEntryPoint": "main",
+            "servertype": "jlink"
+        }
+    ]
+}
```

Comment:
@royratcliffe I think this .vscode/ folder should be included into NuttX, maybe it could be contributed in some other way.